pagerduty/resource_pagerduty_event_orchestration_path_router_test.go

```diff
@@ -213,205 +207,195 @@ func createBaseConfig(t, ep, s, o string) string {
 func testAccCheckPagerDutyEventOrchestrationRouterConfigNoRules(t, ep, s, o string) string {
 	return fmt.Sprintf("%s%s", createBaseConfig(t, ep, s, o),
 		`resource "pagerduty_event_orchestration_router" "router" {
-		parent {
-			id = pagerduty_event_orchestration.orch.id
-		}
-		catch_all {
-			actions {
-				route_to = "unrouted"
+			event_orchestration = pagerduty_event_orchestration.orch.id
```

Comment:
Here and below I replaced `parent {...}` with `event_orchestration = pagerduty_event_orchestration.orch.id` and changed the indentation, that's why it's showing so many changes